Bento/Views/RequiringPreSizingLayoutPass.swift

```diff
@@ -0,0 +1,16 @@
+public protocol RequiringPreSizingLayoutPass {}
```

Comment:
I would rename this to `PreSizingLayoutPassRequiring`, which is more in line with the naming convention for this type of protocol, where the infinitive of the verb comes at the end (eg `HeightCustomizing` 😛 )